agent/src/main/bin/testgrid-agent

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# chkconfig: 345 99 05
+# description: TestGrid Agent daemon script
+#
+# A non-SUSE Linux start/stop script for Java daemons.
+#
+# Set this to Java installation
+JAVA_HOME=/usr/bin/
+
+scriptFile=$(readlink -fn $(type -p $0))             # the absolute, different path of this script file
+scriptDir=$(dirname $scriptFile)                     # absolute path of the script directory
+serviceNameLo="testgridAgent"                        # service name with the first letter in lowercase
```

Comment:
As I've seen, dash is the standard separator in unix world.

Reply:
Fixed with https://github.com/wso2/testgrid/pull/747/commits/b2812b0818ed38457361934f505c931e587efcad